nix/kexec-installer.nix

```diff
@@ -95,7 +95,7 @@ in {
       pkgs.iproute2
     ];
     wantedBy = [ "multi-user.target" ];
-    after = [ "network.target" ];
+    after = [ "network-online.target" ];
```

Comment:
Could this leave the network without static ip addresses when the network-online.target fails i.e. there is no dhcp?

Reply:
maybe. but it worked on the testinstance